core/script_language.cpp

```diff
@@ -214,7 +214,7 @@ void ScriptServer::save_global_classes() {
 		gcarr.push_back(d);
 	}
 
-	ProjectSettings::get_singleton()->set("_global_script_classes", gcarr);
+	ProjectSettings::get_singleton()->set("global/_global_script_classes", gcarr);
```

Comment:
You could make it `global/script_classes` and `global/script_class_icons`.

Reply:
Do you think that is necessary? Since `editor/editor_data.cpp` already sets `global/script_class_icons`.

Reply:
I mean this one should be `global/script_classes`, and the one in `editor_data.cpp` should be `global/script_class_icons`. There's no reason for those two settings to be without a section (the original bug).

Reply:
I should read more thoroughly. Gotcha!